src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -329,11 +329,15 @@ namespace cryptonote
       crypto::public_key key;
       rct::key mask;
       bool unlocked;
+      uint64_t height;
+      crypto::hash txid;
```

Comment:
Which version number should I increment, the major or the minor? Sorry for being noob, this is my very first time to touch the RPC part of Monero:)